eng/Common.props

```diff
@@ -8,7 +8,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <EnableStyleCop>true</EnableStyleCop>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
-    <ExtensionsPackageVersion Condition="'$(ExtensionsPackageVersion)' == ''">2.1.0</ExtensionsPackageVersion>
+    <ExtensionsPackageVersion Condition="'$(ExtensionsPackageVersion)' == ''">8.0.0</ExtensionsPackageVersion>
```

Comment:
I'd prefer not to force all users of this library onto `8.0.0` versions of `Microsoft.Extensions.*`. Upgrading major versions of a transitive dependency is a breaking change to users.